packages/default-theme/components/checkout/steps/ShippingStep.vue

```diff
@@ -32,6 +32,24 @@
               </div>
             </div>
           </template>
+          <template #description="{description}">
+            <div class="sf-radio__description shipping__description">
+              <div class="shipping__delivery">
+                <p>{{ description }}</p>
+              </div>
+              <transition name="sf-fade">
+                <div
+                  v-if="activeShippingMethod === shippingMethod.id"
+                  class="shipping__info"
+                >
+                  <SwPluginSlot
+                    :name="`checkout-shiping-method-${shippingMethod.name}`"
```

Comment:
maybe some global helper that outputs the slot's name by providing area, and extra parameters to have it consistent in whole project. just to remember later on

Reply:
nice idea! Love that!